include/zephyr/net/wifi_mgmt.h

```diff
@@ -356,6 +363,8 @@ struct wifi_connect_req_params {
 	uint8_t bssid[WIFI_MAC_ADDR_LEN];
 	/** Connect timeout in seconds, SYS_FOREVER_MS for no timeout */
 	int timeout;
+	/** Maximum inactivity time in seconds */
+	int32_t max_inactivity;
```

Comment:
Why signed and not `uint32_t`? Would negative value has some special meaning?

Reply:
Type matching  has been kept to make it same as expected in wpa_supplicant. There is no other. 
In case of negative value being passed it will be reported as error.
Ïn supplicant side it's declared as "int ap_max_inactivity;

     /**
         * ap_max_inactivity - Timeout in seconds to detect STA's inactivity
         *
         * */
        int ap_max_inactivity;

Reply:
I don't see any reason for it to be `int` other than coming from the configuration file where there is only `INT` and no `UINT`.

and also `ap_max_inactivity` makes much more sense. And also why is it defined in both `wifi_connect_req_params` and `wifi_ap_config`?